os/board/esp32_devkit/src/esp32_boot.c

```diff
@@ -76,6 +76,11 @@
 #include "esp32_adc.h"
 #include <tinyara/analog/adc.h>
 #endif
+#ifdef CONFIG_SPIRAM_SUPPORT
+extern void esp_spiram_init_cache();
```

Comment:
how about `#include "header-file"`, instead of `extern`?

Reply:
the current esp_idf_port files organization is difficult to include it's header file in os\arch\xtensa, we may refine this later when the organization is modified.